docs/documentation.md

```diff
@@ -105,6 +105,15 @@ If you need to select from a list of static pages in your own backend forms, you
 
 The field's assigned value will be the static page's file name, which can be used to link to the page as described above.
 
+If you need to select from a list of static menus in your own backend forms, you can use the `staticmenupicker` widget:
+
+    fields:
+        field_name:
+            label: Static Menu
+            type: staticmenupicker
+
+The field's assigned value will be the static menu's code, which can be used to link to the menu as described above.
```

Comment:
You would need to provide an example of how someone would use the value, linking to pages based on their code is established already in the docs

Reply:
@LukeTowers If you're still involved with this project, I've added some documentation showing how to retrieve static menu items using the staticMenu code retrieved using the new dropdown form widget.

Reply:
@ericp-mrel No, we all left October and forked it as @wintercms instead, see https://github.com/wintercms/winter/issues/5. We are in the middle of forking all of the RainLab plugins, so if you join our Discord you'll get notified when that's ready (hopefully this upcoming week) and you can resubmit this PR there and we'll most likely merge it)

Reply:
Hey Eric. Looks great, we will review this soon